tests/Endpoints/KeysAndPermissionsTest.php

```diff
@@ -67,8 +50,128 @@ public function testExceptionIfBadKeyProvidedToGetKeys(): void
         $client->getKeys();
     }
 
-    private function getKeys(): array
+    public function testGetKey(): void
```

Comment:
Missing getKeys test

Reply:
No it's the first one `testGetKeysDefault`.
This one try to access `key` route without the `masterKey`.